### DISCUSSION TITLE
просьба

### DISCUSSION THREAD

**Discussion by AleksandrPDA** [Chia-Network/chia-blockchain#8615] [Support]

**Comment by AleksandrPDA** [comment]
Здравствуйте! Можете пожалуйста проверить блок № 897984 который я выиграл 23.09.2021, 08:13 , мой адрес фермера :  xch1689rwj9vtr7dwzmsh6d4qzenjl564cd8vfq59ayp484psxtfvdzqka8lhm .
могу ли я его получить? заранее спасибо!!!  
если потребуется какая ни будь другая информация от меня я сообщу.

Hello! You can please check the block number 897984 which I won on 09/23/2021, 08:13, my farmer's address: xch1689rwj9vtr7dwzmsh6d4qzenjl564cd8vfq59ayp484psxtfvdzqka8lhm.
can i get it? thank you in advance!!!
if any other information is required from me, I will let you know.

**Comment by loppefaaret** [comment]
please go to #8649

**Comment by BrandtH22** [comment]
@AleksandrPDA , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
